demos/ota/ota_demo_core_http/ota_demo_core_http.c

```diff
@@ -1233,63 +1208,57 @@ static void prvMQTTAgentCmdCompleteCallback( CommandContext_t * pxCommandContext
                      eSetValueWithOverwrite );
     }
 }
-
 /*-----------------------------------------------------------*/
 
-static void prvRegisterOTACallback( const char * pTopicFilter,
-                                    uint16_t topicFilterLength )
+static void prvRegisterOTACallback( const char * pcTopicFilter,
+                                    uint16_t usTopicFilterLength )
 {
     bool isMatch = false;
-    MQTTStatus_t mqttStatus = MQTTSuccess;
-    uint16_t index = 0U;
-    uint16_t numTopicFilters = sizeof( otaTopicFilterCallbacks ) / sizeof( OtaTopicFilterCallback_t );
-
-
     bool subscriptionAdded;
```

Comment:
Should these bools also have a prefix? There aren't bools in FreeRTOS so there isn't a precedent.

Reply:
Prefixed bools with 'x'